src/main/kotlin/lotto/DefaultLottoNumber.kt

```diff
@@ -0,0 +1,24 @@
+package lotto
+
+class DefaultLottoNumber private constructor(
+    val numbers: List<Int>,
```

Comment:
```suggestion
    val numbers: Set<Int>,
```
중복된 숫자들을 검증하기 위해 distinct를 활용하셔도 좋습니다만, Set이라는 자료형을 활용하셔도 좋을 것 같습니다. 😅